ballerina/LICENSE

```diff
@@ -12,175 +12,175 @@ At the bottom of this file is a table that shows what each license indicated
 below is and where the actual text of the license can be found.
 
 
-Name                                                                                                Type        License             
+Name                                                                                                Type           License             
 _________________________________________________________________________________________________________________________________________________
 ballerina-command-1.3.15.jar                                                                        jar            apache2             
 commons-text-1.10.0.jar                                                                             bundle         apache2             
 compiler-0.8.9.jar                                                                                  jar            apache2             
-stream-2201.6.1.jar                                                                                 jar            apache2             
-jballerina.java-2201.6.1.jar                                                                        jar            apache2             
-gson-2.8.9.jar                                                                                      jar            apache2             
+future-2201.6.2-20230907-231100-3134cfa2.jar                                                        jar            apache2             
```

Comment:
Shall we remove the timestamps here and the other places?

Reply:
Removed in https://github.com/ballerina-platform/ballerina-distribution/pull/4979/commits/8eaa73fd8613d6b0cd81f5883045036a9c4121f4